baseapp-chats/baseapp_chats/graphql/mutations.py

```
@@ -152,6 +173,136 @@ def mutate_and_get_payload(cls, root, info, profile_id, participants, is_group,
+            return ChatRoomUpdate(
+                errors=[ErrorType(field="image", messages=serializer.errors["image"])]
+            )
+
+        if image is not None:
+            room.image = serializer.validated_data["image"]
+        elif delete_image:
+            room.image = None
+
+        room.save()
+
+        for participant in remove_participants_ids:
```

Ruff (0.8.2)

Loop control variable `participant` not used within loop body

Rename unused `participant` to `_participant`

(B007)